docs/tutorials/prerequisites/install-graphviz.md

```diff
@@ -0,0 +1,9 @@
+## Install Graphviz
+#### On Ubuntu
+```
+sudo apt-get install graphviz
```

Comment:
Can you please move this to the main Installation page? This should go for PIP/VirtualEnvironment and Linux. The Installation script for OS X handles this.

wouldn't pip install graphviz be sufficient?